defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-ferm_rules_defaults:
```

Comment:
You right - but better way to make that is use a compare().
Look my new commit: [https://github.com/ant31-ansible/role-ferm-firewall/pull/6/commits/e9aa72106d3382cc6447f9650f6b3b113dc247c4](url) and [https://github.com/ant31-ansible/role-ferm-firewall/pull/6/commits/49f4439505197b160ec2cbbe5263629170c6622b](url)